charts/velero/templates/_helpers.tpl

```diff
@@ -42,6 +42,31 @@ Create the name of the service account to use for creating or deleting the veler
 {{- end -}}
 {{- end -}}
 
+{{/*
+Generate basic labels
+*/}}
+{{- define "velero.labels" }}
+helm.sh/chart: {{ include "velero.chart" . }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/component: metrics
```

Comment:
`app.kubernetes.io/component: server`

Reply:
The `app.kubernetes.io/name: {{ include "velero.name" . }}` is missing.

Reply:
The `app.kubernetes.io/instance: {{ .Release.Name }}` is missing.

Reply:
@jenting it is in velero.selectorLabels, and included also into velero.labels